Add use-npm CLI flag docs (#17803)
vercel/next.js: docs/api-reference/create-next-app.md

- **-e, --example [name]|[github-url]** - An example to bootstrap the app with. You can use an example name from the [Next.js repo](https://github.com/vercel/next.js/tree/master/examples) or a GitHub URL. The URL can use any branch and/or subdirectory.
- **--example-path [path-to-example]** - In a rare case, your GitHub URL might contain a branch name with a slash (e.g. bug/fix-1) and the path to the example (e.g. foo/bar). In this case, you must specify the path to the example separately: `--example-path foo/bar`

> - **--use-npm** - Explicitly tell the CLI to bootstrap the app using NPM. By default, the CLI will detect if Yarn is installed and use it if it's available.

Reviewer: ```suggestion
- **--use-npm** - Explicitly tell the CLI to bootstrap the app using npm. Yarn will be used by default if it's installed
```
